e2e/tests/ui-driven/src/helpers/addComponent.ts

```diff
@@ -359,6 +364,9 @@ async function createComponentOptions(
     await page.getByPlaceholder("Option").nth(index).fill(option);
     index++;
   }
+
+  await page.getByPlaceholder("Flags (up to one per category)").nth(1).click();
+  await page.getByRole("option", { name: selectedFlag, exact: true }).click();
```

Comment:
Might have missed something but is the `createComponentOptions` function being used anywhere new? Are these lines in the right place?